compose/service.py

```diff
@@ -179,6 +180,7 @@ def __init__(
             ipc_mode=None,
             pid_mode=None,
             default_platform=None,
+            device_requests=None,
```

Comment:
Careful of changing function parameter ordering. It's possible that users are calling this function with positional parameters so it's best to add new parameters to the end of the list (i.e.: after `extra_labels`).